public/services/saved_objects/event_analytics/saved_objects.ts

```diff
@@ -58,6 +58,7 @@ export default class SavedObjects {
     applicationId = '',
     userConfigs = '',
     subType = '',
+    metricType = '',
```

Comment:
Lets consider why we want a separate metricType ?
I think what we want is to invent a new subType, which is what is used to detect promql metric from ppl metric.